src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -200,5 +200,6 @@ public static String formatErrorMsg(String resource) {
             {"R_AKVProviderNull", "The Azure key store provider is null."},
             {"R_objectNullOrEmpty", "The {0} is null or empty."},
             {"R_cekDecryptionFailed", "Failed to decrypt a column encryption key using key store provider: {0}."},
-            {"R_connectTimedOut", "connect timed out"}};
+            {"R_connectTimedOut", "connect timed out"},
+            {"R_InvalidIPAddressPreference", "Invalid option for connection string option IPAddressPreference: {0}"}};
```

Comment:
why is this necessary?

Reply:
To check for correct error message in fail case.

Reply:
Removed and changed as recommended.